tools/ci_build/build.py

```diff
@@ -713,6 +722,9 @@ def use_dev_mode(args):
         return 'OFF'
     if args.use_armnn:
         return 'OFF'
+    if args.use_opencl:
+        #FIXME: hgy, remove later, so that there will be no -Werror in compiling ort
```

Comment:
can this be removed? if not please specify a username instead of initials so it's clear who is responsible for the comment

Reply:
Done